test/e2e/config/digitalocean-ci.yaml

```diff
@@ -33,8 +33,8 @@ providers:
   - name: kubeadm
     type: ControlPlaneProvider
     versions:
-      - name: v1.3.8
-        value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.3.8/control-plane-components.yaml
+      - name: v1.4.5
```

Comment:
should change line 49 as well to `v1.4.99`

Reply:
Fixed